app/utils/pay-links.ts

```diff
@@ -0,0 +1,37 @@
+import { INetwork } from "@app/types/network"
+import { scriptHostname } from "./helper"
+
+// TODO: Return all of these values from the API?
+
+export const getPosUrl = (network: INetwork, address: string): string => {
```

Comment:
Yeah agreed - but we don't currently have the backend setup to return all these things right?  I would suggest we separate out that change in a different PR anyway since it's not related to this work.

Reply:
I tested locally and it didn't work using the shared value.